lib/controls/custom_json_viewer.dart

```diff
@@ -1,7 +1,15 @@
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
+import '../extensions/string_validations.dart';
 import 'package:screensite/theme.dart';
 
+// Make an enum to check for type
+enum SplittingType { UNDERSCORE, CAPITAL, NONE }
+
+var SplittingTypeDict = {
```

Comment:
Let's not create structures that don't solve the current problem. If in the future there will be JSON with another delimiter instead of '_', for instance "=" then we will solve it this way.